docs/UserGuide.md

```diff
@@ -142,8 +142,9 @@ Format: `add n/NAME p/PHONE_NUMBER l/LANGUAGE a/ADDRESS [lv/LAST_VISIT] [v/VISIT
 :bulb: What to include?
 * Adds a new elderly with the following information: `NAME`, `PHONE_NUMBER`, `LANGUAGE`, `ADDRESS`.
 * `LAST_VISIT`, `VISIT` and `HEALTH_CONDITION` may be optionally included.
+* `PHONE_NUMBER` must be a 8 digit numeric value, and there should not be country code
```

Comment:
This is slightly different from the phone number specified under the section "structure of an elderly contact", maybe you want to update that as well to make it consistent?

Reply:
Noted.